index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Anika's Portfolio</title>
+    <link href="styles/style.css" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h1>Anika's Personal Portfolio</h1>
+        <nav>
+            <ul>
+                <li>
+                    <a href="/pages/about.html">About Me</a>
+                </li>
+                <li>
+                    <a href="/pages/portfolio.html">Projects</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+        <section id="welcome-message">
+            <h2>Welcome!</h2>
+            <p> 
+                Thank you for visiting my online portfolio!
+            </p>
+            <div id="grid-container">
```

Comment:
Interesting! Good practice working on styling a grid and then applying different styles to each little box! 

That seems like a tedious job.

In practice, you'd probably grab an image of the logo and use the image tag for it since that's more semantic, but I think you did this for practice with css grid.